FAIR_eva/README.md

```diff
@@ -1,12 +1,12 @@
 # /fair_eva/
-This [W3ID](https://w3id.org) provides a persistent URI namespace for FAIR EVA: Evaluator, Validator and Advisor, a FAIRness assesment tool available at https://github.com/EOSC-synergy/FAIR_eva
+This [W3ID](https://w3id.org) provides a persistent URI namespace for FAIR EVA: Evaluator, Validator and Advisor, a FAIRness assessment tool available at https://github.com/EOSC-synergy/FAIR_eva
 
 ## Contact
-This space is administered by:  
+This space is administered by:
 
-**Fernando Aguilar**  
-*Postdoctoral Researcher - CSIC*  
-Santander, Spain  
-<aguilarf@ifca.unican.es>  
+**Fernando Aguilar**
```

Comment:
The trailing spaces were there to force line breaks for formatting reasons.  Otherwise this is all one line.  Also consider making these lines a list with `- ` or `* ` prefixes.